### DISCUSSION TITLE
-AllowRebootPassThru

### DISCUSSION THREAD

**Discussion by Furlooze** [PSAppDeployToolkit/PSAppDeployToolkit#957] [Q&A]
Hello all !

Question from a PS AppDT newbie 
For everybody it seems the -AllowRebootPassThru do the trick ... but not for me . I probably miss something 
Context : custom scripts, no .MSI , [Int32]$mainExitCode = 3010 (expected behavior)
If I do not use -AllowRebootPassThru, deployment status show error "Deployment not detected after installation" : normal behavior 

When adding argument
Deploy-Application.PS1 -DeploymentType 'Install' -DeployMode 'NonInteractive' -AllowRebootPassThru

Exit Code 3010 is correctly passed into .log, but SCCM deployment status is failed : "incorrect function" ? 
![image](https://github.com/PSAppDeployToolkit/PSAppDeployToolkit/assets/124881159/af1a4d46-06d0-4e9b-9415-5502d909259d)
![image](https://github.com/PSAppDeployToolkit/PSAppDeployToolkit/assets/124881159/faa0127f-ec3b-4ffd-a06e-819de279a3be)
![image](https://github.com/PSAppDeployToolkit/PSAppDeployToolkit/assets/124881159/e955e4df-03d3-4c7d-b19f-bd474b708e30)

Not a big deal, after a reboot & the next App Eval. , status is OK (detection method fulfilled) . But I would like to understand why & also differentiate at first sight "failed" & "pending reboot" in my deployment status . 

Thx !

**Comment by DanGough** [comment]
Do you get the same result using Deploy-Application.exe?

**Comment by mjr4077au** [comment]
Closing due to no response.
